### ISSUE TITLE
Specify "to_bytes" and "from_bytes"

### ISSUE THREAD

**Issue by mratsim** [ethereum/eth2.0-specs#1395]
In https://github.com/ethereum/eth2.0-specs/blob/v0.8.3/specs/core/0_beacon-chain.md#int_to_bytes

The int_to_bytes and bytes_to_int depend on Python implementation of to_bytes

```python
def int_to_bytes(n: uint64, length: uint64) -> bytes:
    """
    Return the ``length``-byte serialization of ``n``.
    """
    return n.to_bytes(length, ENDIANNESS)
```

A description of what this method does should be added so that we don't have to look into the Python spec.

**Comment by djrtwo** [comment]
I'll handle this in the `dev` branch. Seems reasonable to add another line to the header comment to clarify

**Comment by protolambda** [comment]
Fixed in #1401
